src/components/Editor/Editor.css

```diff
@@ -98,7 +98,7 @@ html[dir="rtl"] .editor-mount {
 }
 
 .highlight-line .CodeMirror-line {
-  animation: fade-highlight-out 1.5s normal forwards;
+  animation: fade-highlight-out 4s normal forwards;
```

Comment:
hmm, i think we *only* want to do this when we're navigating from the console.

Perhaps:

```css
.highlight-line.ong .CodeMirror-line  { 
  animation: fade-highlight-out 3s normal forwards;
}
```